packages/terra-application/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Updated error boundary jest snapshots.
```

Comment:
```suggestion
  * Updated error boundary jest snapshots.
  * Updated props table WDIO screenshots
```

Reply:
Changed in https://github.com/cerner/terra-application/pull/319/commits/d02a146191845c75c9e91fb8eb35da975c9c8a76